packages/server/src/integrations/googlesheets.ts

```diff
@@ -626,7 +626,7 @@ export class GoogleSheetsIntegration implements DatasourcePlus {
     sheet: string
     rowIndex: number
     row: any
-    table: Table
+    table?: Table
```

Comment:
This was the root of the problem: `table` doesn't appear to always be passed (ever be passed?). So I've handled it not being there. Not sure if it's ever passed to be honest, and whether it should be removed entirely.

The typing on `IntegrationBase` is quite bad, lots of anys, allowing this sort of thing through.